csvcut.py

```diff
@@ -0,0 +1,104 @@
+from argparse import ArgumentParser
+import sys
+
+DESCRIPTION = 'csvcut - selects the specified fields from cvs file'
+EXAMPLES = 'csvcut -f 1,2 stat.txt | less -SR'
+
+def keep_fields(ind, row, output_stream):
+    """
+    Prints the specified fields from .csv file
+    
+    :param indexes: indexes of the required fields represented as a list of indexes
+    :param row: row represented as a list of columns
+    :param output_stream: a stream to print the row with required columns
+    """
+    output_line = ''
+    for i, column in enumerate(row):
+        if i in ind:
```

Comment:
It will be better if you filter the fields you need and do ','.join(fields). Also, separator option should be taken into account for the output file